crates/containerd-shim-wasm/src/sandbox/oci.rs

```diff
@@ -4,13 +4,35 @@ use std::collections::HashMap;
 use std::io::{ErrorKind, Write};
 #[cfg(unix)]
 use std::os::unix::process::CommandExt;
+use std::path::PathBuf;
 use std::process;
 
 use anyhow::Context;
 pub use oci_spec::runtime::Spec;
 
 use super::error::Result;
 
+pub fn load_from_bundle(bundle: PathBuf) -> Result<Spec> {
+    let mut spec = Spec::load(bundle.join("config.json").to_str().unwrap())?;
+    spec.canonicalize_rootfs(&bundle)?;
+    Ok(spec)
+}
+
+pub fn get_oci_artifact(spec: &Spec) -> Option<String> {
+    match spec.annotations().clone() {
+        Some(annotations)
+            if annotations
+                .contains_key("application/vnd.bytecodealliance.wasm.component.layer.v0+wasm") =>
```

Comment:
Is there a canonical location for this in another library or should I move this to a const some where in the project.

Reply:
put it in the oci package for now